### PR TITLE
Update progress bar

### LINKED ISSUE #26: Cannot see details of pending actions
When the description is long it the information is cut off and there is no way to view the full description (like you would with the detail view of the voting app). 

image

If we want to maintain the delay app, we should probably replicate the same functionality of the voting app (filtering by app, filtering by status), and add a detail view which can show full more information that can fit on a card.

### LINKED ISSUE #27: App icon in delay app card seems to be mis-formatted
example:

image

### LINKED ISSUE #28: Avoid hover states for accessibility
In the delay app we show a progress bar and if you hover over you see the actual time remaining. I think we should go ahead and make the time remaining always visible as there is a general pattern of avoiding hover states in the aragonDS for accessibility reasons. 

image

### PR DESCRIPTION
**Pull request by fabriziovigevani**
closes #26 , closes #27 , closes #28 

- Added Detail view for Delays.

- Removed timer in the Delay card and only show progress bar (time remaining can be seen in the detailed view).

- Added filters by Status (ongoing, paused, pending for execution) and App.

TODO: 

- [ ] Update user-guide gifs.
- [ ] Add deeplinking

### CODE REVIEW

File: app/src/app-state-reducer.js [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
-const compareScripts = (s1, s2) => {
-  const diff = Boolean(s1.pausedAt) - Boolean(s2.pausedAt)
+const compareDelayedScripts = (ds1, ds2) => {
+  const diff = Boolean(ds1.pausedAt) - Boolean(ds2.pausedAt)
   if (diff !== 0) return diff
 
-  const timeRemainingS1 = s1.executionTime - s1.pausedAt
-  const timeRemainingS2 = s2.executionTime - s2.pausedAt
+  const timeRemainingDS1 = ds1.executionTime - ds1.pausedAt
```
Comment (rperez89) [inline app/src/app-state-reducer.js]: I would try to not  abbreviate variable names, it took me some time to realize that ds is delayed script

File: app/src/app-state-reducer.js [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+  const timeRemainingDS2 = ds2.executionTime - ds2.pausedAt
```
Comment (rperez89) [inline app/src/app-state-reducer.js]: same here for ds1 and 2

File: app/src/components/DelayActions.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import ResumeIcon from '../assets/resume.svg'
+import PauseIcon from '../assets/pause.svg'
+
+import STATUS from '../delay-status-types'
+
+const getMainActionProps = (status, theme) => {
+  switch (status) {
+    case STATUS.ONGOING:
+      return {
+        mode: 'normal',
```
Comment (rperez89) [inline app/src/components/DelayActions.js]: just for consistency, i would start the strings all with uppercase or lowercase

File: app/src/components/EmptyFilteredDelays.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+          css={`
+            max-width: 270px;
+            text-align: center;
+            margin-top: ${1 * GU}px;
+            margin-bottom: ${4 * GU}px;
+            color: ${theme.surfaceContentSecondary};
+            font-size: 16px;
+          `}
+        >
+          We can’t find any item matching your filter selection.{' '}
```
Comment (rperez89) [inline app/src/components/EmptyFilteredDelays.js]: is that {' '} ok?

File: app/src/components/ScriptText.js [hunk 5]
```diff
@@ -31,11 +31,7 @@ const ScriptText = React.memo(
                 isAddress ? (
                   <span title={part} key={index}>
                     {' '}
-                    <LocalIdentityBadge
-                      badgeOnly={disabled}
-                      compact
-                      entity={part}
-                    />{' '}
+                    <LocalIdentityBadge badgeOnly={disabled} compact entity={part} />{' '}
```
Comment (rperez89) [inline app/src/components/ScriptText.js]: Same here is that {' '} ok?

File: app/src/hooks/useFilterDelays.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+function testStatusFilter(filter, scriptStatus) {
+  return (
+    filter === NULL_FILTER_STATE ||
+    (filter === STATUS_FILTER_ONGOING && scriptStatus === STATUS.ONGOING) ||
+    (filter === STATUS_FILTER_PAUSED && scriptStatus === STATUS.PAUSED) ||
+    (filter === STATUS_FILTER_PENDING && scriptStatus === STATUS.PENDING_EXECUTION)
+  )
+}
+
+function testAppFilter(filter, script, { apps, thisAddress }) {
```
Comment (rperez89) [inline app/src/hooks/useFilterDelays.js]: should we call that function as testAppFilter?

File: app/src/hooks/useFilterDelays.js [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+  const { executionTargets } = script
+
+  if (filter === NULL_FILTER_STATE) {
+    return true
+  }
+  if (filter === APP_FILTER_THIS) {
+    return executionTargets.length === 0 || executionTargets.includes(thisAddress)
```
Comment (rperez89) [inline app/src/hooks/useFilterDelays.js]: i would rename that thisAddress variable to something that makes more sense

File: app/src/screens/DelayDetail.js [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+          </Box>
+        }
+      />
+    </React.Fragment>
+  )
+})
+
+function Status({ delay }) {
+  const { status, executionTime, pausedAt } = delay
+  const ONE_SECOND = 1000
```
Comment (rperez89) [inline app/src/screens/DelayDetail.js]: Maybe ONE_SECOND_IN_MS ?

File: app/src/screens/Delays.js [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+              selectDelay={selectDelay}
+            />
+          )}
+        </React.Fragment>
+      </React.Fragment>
+    )
+  }
+)
+
+const ThisDelay = ({ showTag }) => (
```
Comment (rperez89) [inline app/src/screens/Delays.js]: does it make sense to rename this to something like currentDelay? also i think that we should keep const, variable, and function names starting with lowercase

File: app/src/screens/Delays.js [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+          margin-left: ${1 * GU}px;
+        `}
+      >
+        this app
+      </Tag>
+    )}
+  </div>
+)
+
+const DelayGroups = ({ ongoingDelays, pausedDelays, pendingDelays, selectDelay }) => {
```
Comment (rperez89) [inline app/src/screens/Delays.js]: same here

File: app/src/screens/Delays.js [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+              })}
+            </CardLayout>
+          </section>
+        ) : null
+      )}
+    </React.Fragment>
+  )
+}
+
+const GroupName = ({ title, count, compactMode }) => {
```
Comment (rperez89) [inline app/src/screens/Delays.js]: same lowercase here

File: app/src/components/DelayActions.js (hunk 3 above)
Comment (fabriziovigevani) [reply app/src/components/DelayActions.js]: The thing is that some of the values are Components which by convention start with Upper case, and the labels are needed to start with uppercase as well.

File: app/src/hooks/useFilterDelays.js (hunk 6 above)
Comment (fabriziovigevani) [reply app/src/hooks/useFilterDelays.js]: Will change to `checkAppFilter` (same for `testStatusFilter`)

File: app/src/screens/Delays.js (hunk 9 above)
Comment (fabriziovigevani) [reply app/src/screens/Delays.js]: For the renaming i think it makes sense to rename it, though i think a better description could be `ThisApp` as it references the Delay app.

Regarding the lowercase, this are all React components and by convention they are named with Uppercase.

Comment (rperez89) [reply app/src/screens/Delays.js]: ohh didn't notice that they were react components, seems good

**Pull request merged**